.circleci/config.yml

```diff
@@ -116,6 +116,16 @@ jobs:
           name: Validate Validator-Sets Configs
           command: ./scripts/circle_validate_configs.sh
       - build_teardown
+  validate-cluster-test-dockerfile:
+    description: Validate that committed docker files for cluster test are up to date
+    executor: build-executor
+    steps:
+      - build_setup
+      - run:
+          # generate dockerfile; diff config to existing one
+          name: Validate cluster-test dockerfiles
+          command: ./docker/cluster-test/generate.sh && git diff --exit-code
```

Comment:
We can skip the `git diff --exit-code`.  The `build_teardown` step below has a check to screen changed files and will error out if the check fails.